src/AppInstallerSharedLib/Errors.cpp

```diff
@@ -262,6 +262,73 @@ namespace AppInstaller
                 return "Invalid parameter.";
             case APPINSTALLER_CLI_ERROR_INSTALL_SYSTEM_NOT_SUPPORTED:
                 return "Package not supported by the system.";
+            
+            // Status values for check package installed status results.
+            case WINGET_INSTALLED_STATUS_ARP_ENTRY_NOT_FOUND:
+                return "The Add & Remove Programs Entry for the package could not be found.";
+            case WINGET_INSTALLED_STATUS_INSTALL_LOCATION_NOT_APPLICABLE:
+                return "The install location is not applicable.";
+            case WINGET_INSTALLED_STATUS_INSTALL_LOCATION_NOT_FOUND:
+                return "The install location could not be found.";
+            case WINGET_INSTALLED_STATUS_FILE_HASH_MISMATCH:
+                return "The hash of the existing file did not match. Please uninstall and try again.";

```

Comment:
```suggestion
                return "The hash of the existing file did not match.";

```
This is a status check of one file for callers like Intune to notify IT admins of package status. IT admins would decide what to do. Maybe the package is updated outside of winget etc.